### DISCUSSION TITLE
Best model for Deforum??

### DISCUSSION THREAD

**Discussion by VrGameDevGirl1984** [civitai/civitai#526] [Q&A]
So far I can't seem to get anything off this site to work with Deform. I'm running it locally. Please help.

**Comment by jaysenttrue** [comment]
Have you set it up on visual studio?

**Comment by zephiowl** [comment]
> So far I can't seem to get anything off this site to work with Deform. I'm running it locally. Please help.

How are you running SD? I've been able to get some custom models working fine with automatic1111 and the deform plugin. Do you have any error logs, codes or information to share?
